Added note and updated text (#3131)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/allow-or-prevent-custom-script.md

    Set-PnPSite <SiteURL> -NoScriptSite $false
    ``` 

If you change this setting for a classic team site, it will be overridden by the Custom Script setting in the admin center within 24 hours.
> [!NOTE]

>     > Allow/disallow of custom script to individual OneDrive’s cannot be done.

Reviewer: ```suggestion
> You cannot allow or prevent custom scripts to an individual user's OneDrive.
```
